data/brands/shop/hairdresser.json

```
@@ -33,6 +33,18 @@
     }
   },
   "items": [
+     {
+      "displayName": "Hot Pepper Beauty",
+      "locationSet": {"include": ["jp"]},
+      "tags": {
+        "brand": "Hot Pepper Beauty",
+        "brand:wikidata": "Q123029328",
+        "name:en": "Hot Pepper Beauty",
+        "shop": "hairdresser",
+        "name:ja": "ホットペッパービューティーなら",
```

According to their website and Facebook page, this is their name in Japanese
```suggestion
        "name:ja": "ホットペッパービューティー",
```